app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+  after_save :update_likes_counter
+
+  def update_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
- [ ] OPTIONAL
Following best practices `update_like_counter` should be marked as _private_ after implementing the `after_save` method.